frontend/public/components/pdb/pdb-models.ts

```diff
@@ -0,0 +1,38 @@
+import { MatchLabels } from '@console/dynamic-plugin-sdk/src/api/common-types';
+import { PodDisruptionBudgetKind } from '../../module/k8s';
+import { PodDisruptionBudgetModel } from '../../models';
+
+export const convertToInt = (val) => {
+  if (Number.isInteger(val)) {
+    return val;
+  }
+  return val?.includes('%') ? val : parseInt(val, 10);
```

Comment:
How does this behave if the string isn't a valid number?

Reply:
I think Sam is onto something here that you need to specially handle @cyril-ui-developer 

Example:

I input `asdf` into the YAML:
![Screen Shot 2022-04-08 at 2 10 23 PM](https://user-images.githubusercontent.com/8126518/162498117-aada38b2-2be6-4f67-8167-f0a825ac95cb.png)

See it in my Form (not ideal as I cannot edit it):
![Screen Shot 2022-04-08 at 2 10 29 PM](https://user-images.githubusercontent.com/8126518/162498118-2afd3765-c749-4706-91b5-f3562b15d060.png)

(I needed to make a change to my name to make the form convert back to YAML) When returning to YAML after the parsing you get `.nan`:
![Screen Shot 2022-04-08 at 2 10 42 PM](https://user-images.githubusercontent.com/8126518/162498120-e62c95dc-6e19-4870-bb92-7e5fba56b4af.png)